web/help/GuideUnit1.md

```diff
@@ -504,9 +533,10 @@ more hints, expand the sections below for an example and explanation.
     solidCircle :: Number -> Picture
     ~~~~~
 
-    This type signature tells you that the only argument to `solidCircle`
-    is a number, and the result is a picture.  Here's an example of a
-    program that uses `solidCircle`.
+    This type signature tells you that the only argument of `solidCircle`
+    is a number and the result is a picture.  
```

Comment:
Needs either a comma before and, or (preferred) "and **that** the result is a picture."